pkg/trace/api/debugger_test.go

```diff
@@ -0,0 +1,137 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
I realise you've copied these tests from the other proxy. That's fine. But I was wondering whether there would be a way to easily de-dupe them by perhaps creating a function which runs the tests and takes as input a config and an `http.ReverseProxy` ?

If it's complicated or not an easy approach, it's fine to leave it as it is.

Reply:
Hm, I'm not sure it's worth the effort or feasible at all. The tests only resemble each other in structure. But they differ in the tested types and in the assertions.